staging/src/k8s.io/kubectl/go.sum

```diff
@@ -322,6 +322,7 @@ github.com/mitchellh/gox v0.4.0/go.mod h1:Sd9lOJ0+aimLBi73mGofS1ycjY8lL3uZM3JPS4
 github.com/mitchellh/iochan v1.0.0/go.mod h1:JwYml1nuB7xOzsp52dPpHFffvOCDupsG0QubkSMEySY=
 github.com/mitchellh/mapstructure v0.0.0-20160808181253-ca63d7c062ee/go.mod h1:FVVH3fgwuzCH5S8UJGiWEs2h04kUh9fWfEaFds41c1Y=
 github.com/mitchellh/mapstructure v1.1.2/go.mod h1:FVVH3fgwuzCH5S8UJGiWEs2h04kUh9fWfEaFds41c1Y=
+github.com/moby/ipvs v1.0.1/go.mod h1:2pngiyseZbIKXNv7hsKj3O9UEz30c53MT9005gt2hxQ=
```

Comment:
why is kubectl affected?

Reply:
No idea, maybe this is something related to vendor-update. Let me remove this line, and see what happens with verify-dependencies again here.

Reply:
So, if I remove those lines and run it complains. Re-generating go.mod from kubectl only vendors this again. Let me see what library is indirect vendoring this

Reply:
Hum, so apparently because ipvs is vendored in component-helpers ipvs gets vendored as well...strange, I'll do all of this again and see if I've messed with something